### ISSUE TITLE
aarch64 migration

### ISSUE THREAD

**Issue by MarkWieczorek** [conda-forge/pyshtools-feedstock#56]
### Comment:

I would like to migrate the pyshtools project to aarch64. I added pyshtools to the migration list [here](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/arch_rebuild.txt), however, after 6 days, nothing has happened at all. Conda packages already exist for all of the pyshtools dependencies on aarch64 (at least on my machine), so I don't think that this is because of the lack of a compatible dependency.

@conda-forge/arm-arch, could you re-run the arch migration bot for this project? Or give me some guidance on what needs to be done to get this rolling?

Thanks!

**Comment by hmaarrfk** [comment]
It seems that from here: https://conda-forge.org/status/migration/aarch64andppc64leaddition

if you click on the awaiting parents, you see that you are "waiting on pyqt" to get migrated.

There are about 0 chance of the pyqt package gets migrted.

So I think it is one of your dependencies that is pulling it in...... but, i just built an environment locally on linux, and I can't find the dependency that is getting qt pulled in.

Maybe it is `jupyter`. You can try replacing that with `jupyter_core`.

Sometime the bot gets stuck. Especially if you have had a large refactor recently.

You can try to make a pull request and add the aarch and ppc64le lines manually:
https://github.com/conda-forge/opencv-feedstock/blob/main/conda-forge.yml#L1C1-L3C26

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
Yeah, my hunch is that it was `jupyter` that was pulling in the qt stuff.

it pulls in:
```
  + jupyterlab                              4.1.6  pyhd8ed1ab_0     conda-forge        8MB
  + notebook                                7.1.3  pyhd8ed1ab_0     conda-forge        4MB
  + jupyter                                 1.0.0  pyhd8ed1ab_10    conda-forge        9kB
```
Locally, I am able to see that it pulls in the qt stuff too. Maybe it isn't `jupyter_core` that you need to depend on but something else.
